### PR TITLE
[SD-1455] OpenAPI docs for Cart attributes are missing example values and response example

### PR DESCRIPTION
**Pull request by pmilewski**

### CODE REVIEW

File: api/docs/v2/storefront/index.yaml [hunk 1]
```diff
@@ -550,7 +550,62 @@ paths:
...
-              examples: {}
+              examples:
+                example:
+                  value:
+                    data:
+                      id: '511'
+                      type: cart
+                      attributes:
+                        number: R904373524
+                        item_total: '0.0'
```
Comment (damianlegawiec) [inline api/docs/v2/storefront/index.yaml:561]: @pmilewski let's have some positive numbers here also let's include the includes with one for each type - line item, shipment, promotion, payment, addresses, etc

**Review by damianlegawiec — COMMENTED** [review]
@MatthewKennedy could we extend this to include shipment, promotion and a tax adjustment?

**Comment by MatthewKennedy** [comment]
@damianlegawiec , Sure yes, it shouldn't be a problem.

**Comment by MatthewKennedy** [comment]
Need To Add Payment to includes still....

**Comment by MatthewKennedy** [comment]
Added payment to includes and added response examples to endpoints that return the cart.

**Comment by MatthewKennedy** [comment]
Some more changes to make ...

**Comment by MatthewKennedy** [comment]
@damianlegawiec That should do it for the cart schema and response examples.

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**